setup.cfg

```
@@ -21,6 +21,5 @@ classifiers = Development Status :: 3 - Alpha
 include_package_data = True
 zip_safe = True
 packages = find:
-install_requires = numpy
-                   scikit-learn>="0.24.0"
-
+install_requires = ["numpy",
+                   "scikit-learn>=0.24.0"]
```

I typically use this syntax, does it not work?

```suggestion
install_requires = 
    numpy
    scikit-learn>=0.24.0
```